Jenkinsfile

```diff
@@ -387,7 +387,13 @@ ansiColor('xterm') {
                   }
 
                   try {
-                    sh script: "npm run lerna --silent -- publish --skip-npm --skip-git  --yes --repo-version=${version}"
+                    def publishScript = "npm run lerna --silent -- publish --skip-npm --skip-git  --yes --repo-version=${version}"
```

Comment:
Just in case lerna overwrites the exact version numbers in the deps.js generated package.json, we should probably add `--exact` to the publish call here.